### PR TITLE
Integrate guacscanner

### PR DESCRIPTION
**Pull request by jsf9k**
## 🗣 Description ##

This pull request integrates [cisagov/guacscanner-docker](https://github.com/cisagov/guacscanner-docker) into this Docker composition.

## 💭 Motivation and context ##

The functionality of [cisagov/guacscanner](https://github.com/cisagov/guacscanner) must be integrated into this Docker composition for the RPT UAT taking place in the COOL.

See also:
- cisagov/guacscanner/pull/1
- cisagov/guacscanner-docker#2
- cisagov/ansible-role-guacamole#38
- cisagov/guacamole-packer#53
- cisagov/cool-assessment-terraform#149

## 🧪 Testing ##

I have applied these changes to env0 in our staging COOL environment and verified that they function as expected.

## ✅ Checklist ##

- [x] This PR has an informative and human-readable title.
- [x] Changes are limited to a single goal - _eschew scope creep!_
- [x] All relevant type-of-change labels have been added.
- [x] I have read the [CONTRIBUTING](../blob/develop/CONTRIBUTING.md) document.
- [x] These code changes follow [cisagov code standards](https://github.com/cisagov/development-guide).
- [x] All relevant repo and/or project documentation has been updated to reflect the changes in this PR.
- [x] Tests have been added and/or modified to cover the changes in this PR.
- [x] All new and existing tests pass.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,9 @@ remote desktop gateway.
 - [guacd](https://hub.docker.com/r/guacamole/guacd/) server-side proxy for
 Guacamole.
 - [Postgres](https://hub.docker.com/_/postgres/) relational database.
+- [cisagov/guacscanner-docker](cisagov/guacscanner-docker) utility for
```
Comment (dav3r) [inline README.md]: Are you sure that this URL will work correctly when this is merged?  When I click on it in the current branch, it does not:
https://github.com/cisagov/guacamole-composition/tree/improvement/support-guacscanner

File: README.md [hunk 2]
```diff
@@ -11,6 +11,9 @@ remote desktop gateway.
...
+  continually scanning the instances in an AWS VPC and updating the
```
Comment (dav3r) [inline README.md]:
```suggestion
  continually scanning the EC2 instances in an AWS VPC and updating the
```

File: README.md [hunk 3]
```diff
@@ -51,6 +54,11 @@ composition on a publicly-accessible host:
 |----------|---------|
 | postgres-username | Text file containing the username of the `postgres` user used by the `guacamole` container |
 | postgres-password | Text file containing the password of the `postgres` user used by the `guacamole` container |
+| private_ssh_key | Text file containing the private ssh key to use for SFTP file transfer in Guacamole. |
```
Comment (dav3r) [inline README.md]: Capitalization:
```suggestion
| private_ssh_key | Text file containing the private SSH key to use for SFTP file transfer in Guacamole. |
```

**Review by dav3r — APPROVED** [review]
LGTM, aside from one question and two tiny requested changes. 🥑

File: README.md (hunk 1 above)
Comment (jsf9k) [reply README.md]: No, that is a mistake.  Please see commit 4ee4b76.

Comment (dav3r) [reply README.md]: Thanks - wasn't sure if that was a GitHub shortcut there or not.

**Review by mcdonnnj — APPROVED** [review]
✨

**Pull request merged**